ci-operator/step-registry/ibmcloud/deprovision/byok/ibmcloud-deprovision-byok-commands.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# IBM Cloud CLI login
+function ibmcloud_login {
+    export IBMCLOUD_CLI=ibmcloud
+    export IBMCLOUD_HOME=/output
+    region="${LEASED_RESOURCE}"
+    rg=$1
+    export region
+    echo "Try to login to ${rg}..."
```

Comment:
Insert "${IBMCLOUD_CLI}" config --check-version=false before login, to avoid ibmcloud plugin check.